src/Reducer.js

```diff
@@ -48,6 +48,11 @@ function inject(state, action, props, scenes) {
             case POP_ACTION:
                 assert(!state.tabs, "pop() operation cannot be run on tab bar (tabs=true)")
                 return {...state, index:state.index-1, children:state.children.slice(0, -1) };
+            case POP_TO_TOP:
+                assert(state.tabs, "Parent="+state.key+" is not tab bar, popToTop action is not valid");
+                let child = state.children[state.index];
+                let newChild = {...child, ...{children: [child.children[0]], index:0}};
+                return {...state, ...{children:[...state.children.slice(0,state.index), newChild, ...state.children.slice(state.index+1)]}};
```

Comment:
I'm not sure how that would look for a general case.. I have no type reset on the child, its being forced into the reducer at the moment. It could be the default behaviour of the tabs since it's also done that way in iOS while Android doesnt use tabs..